### PR TITLE
Plugins update

### PR DESCRIPTION
**Pull request by hypervillain**
Merge during plugins release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
   "devDependencies": {
     "@nuxtjs/prismic": "^1.4.2",
     "@prismicio/slice-simulator-vue": "^0.2.3",
+    "@slicemachine/adapter-nuxt2": "^0.0.2-dev-plugins-m3.0",
```
Comment (lihbr) [inline package.json:22]: Can you align to 3.4 also?

**Review by lihbr — APPROVED** [review]
LGTM!

File: package.json (hunk 1 above)
Comment (lihbr) [reply package.json:22]: Actually since many other alpha will be released, we don't care for now

**Comment by hypervillain** [comment]
Thanks @lihbr :)

**Comment by hypervillain** [comment]
@lihbr do we still need to transpile @prismicio/vue?

**Pull request merged**